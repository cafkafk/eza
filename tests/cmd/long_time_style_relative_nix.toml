bin.name = "eza"
args = "tests/itest --long --time-style relative"
