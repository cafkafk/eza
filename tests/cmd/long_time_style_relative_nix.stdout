.rw-r--r--  0 nixbld 53 years a
.rw-r--r--  0 nixbld 53 years b
.rw-r--r--  0 nixbld 53 years c
.rw-r--r--  0 nixbld 53 years d
.rw-r--r--  0 nixbld 53 years e
drwxr-xr-x  - nixbld 53 years exa
.rw-r--r--  0 nixbld 53 years f
.rw-r--r--  0 nixbld 53 years g
.rw-r--r--  0 nixbld 53 years h
.rw-r--r--  0 nixbld 53 years i
.rw-r--r--  0 nixbld 53 years image.jpg.img.c.rs.log.png
.rw-r--r-- 19 nixbld 53 years index.svg
.rw-r--r--  0 nixbld 53 years j
.rw-r--r--  0 nixbld 53 years k
.rw-r--r--  0 nixbld 53 years l
.rw-r--r--  0 nixbld 53 years m
.rw-r--r--  0 nixbld 53 years n
.rw-r--r--  0 nixbld 53 years o
.rw-r--r--  0 nixbld 53 years p
.rw-r--r--  0 nixbld 53 years q
drwxr-xr-x  - nixbld 53 years vagrant
